### ISSUE TITLE
Create HomePage

### ISSUE THREAD

**Issue by ashish11chawda** [we-developers-community/community-website#1]
Contents of HomePage
- Navigation Bar
     - Resources
     - Events
     - Contributors
     - Blog(alpha)
- Developers Community's Logo
- motto
- Featured Events
- Resources Page Overview
- Contributors
- Footer

**Comment by deepaksy** [comment]
# `Can you please elaborate what do you mean by resource page Overview.`

**Comment by ashish11chawda** [comment]
@deepaksy 
Yeah, this section of HomePage will show how the resources page can be helpful for a computer science student

So a brief overview as a picture or a single card can be useful

Instead of a card we can do carousel also

Whichever is good

**Comment by nishantpathak27** [comment]
@pixan198 Please allow me to start working on the home page.

**Comment by ashish11chawda** [comment]
@NishantPathak2704 go ahead bro
Just discuss the mock-up with other members

**Comment by vaibhavscor** [comment]
> @NishantPathak2704 go ahead bro
> Just discuss the mock-up with other members

Sir can I also start with home page ?

**Comment by ashish11chawda** [comment]
Sure bro @vaibhavscor 
Teamup with @NishantPathak2704
